### DISCUSSION TITLE
opt folder

### DISCUSSION THREAD

**Discussion by benmaxyoung** [beeware/mobile-forge#23] [Q&A]
I would like to ask a question. Are the iOS static libraries and header files related to litesync placed in the lib and include folders of `recipes/litesync/opt/`?

**Comment by freakboy3742** [comment]
No. The recipe folder should only contain the recipe configuration, any patch files needed to compile the package, and the test files (although the test files aren't currently used).

If litesync requires some external headers to build, that implies there's a non-Python dependency. Those dependencies need to be packaged as a separate recipe (see, for example libjpeg and Pillow).

If litesync *provides* headers, they should be in the source tarball; if they're needed for other packages that depend on litesync, they should be in the wheel that is produced - but they should already be packaged like that.
